monailabel/tasks/infer/basic_infer.py

```diff
@@ -57,7 +57,7 @@ def __init__(
         output_label_key: str = "pred",
         output_json_key: str = "result",
         config: Union[None, Dict[str, Any]] = None,
-        load_strict: bool = False,
+        load_strict: bool = True,
```

Comment:
```suggestion
        load_strict: bool = False,
```

Reply:
Then only the warning?
My idea behind that was that this makes adding new models harder. Which is why I proposed changing the default to True and setting the value to False in all the existing models. 
In my case the network would not load any weights at all but MONAILabel would not tell me. The state dict was completely different but I did not even get a warning. I was thus wondering for a few hours why a perfectly fine network wouldn't work in MONAILabel...

Reply:
In that case can you verify all sample apps..

Reply:
@SachidanandAlle Can you explain what you mean?

Reply:
we need to make sure existing pre-trained models can load with this flag. 
radiology/pathology and endoscopy apps/models

Reply:
@SachidanandAlle Absolutely right. What I had planned was to add that flag in the step, when you guys approve of the changes, which I was not sure of.
However you already merged the branch now. Shall I still add the `flag=False` for all existing models or do you want to do that based on feedback what breaks?

Thanks for the quick response!

Reply:
I will add that today and update this PR / open another one if I can't do that. It might break some preexisting code otherwise.